internal/core/application/operator_service.go

```diff
@@ -2052,21 +2056,28 @@ func (o *operatorService) splitMarketFragmenterFunds(
 		}
 	}
 
-	if assetValuePair.baseValue == 0 {
-		chRes <- FragmenterSplitFundsReply{
-			Err: fmt.Errorf(
-				"missing base asset funds",
-			),
+	// If the market has zero balance, it's mandatory to send both base and quote
+	// asset to the market fragmenter account. Otherwise, it's possible to split
+	// funds of only one of the assets of the pair.
+	marketBalance, _ := getBalanceForMarket(o.repoManager, ctx, market, false)
```

Comment:
maybe lets make this query AFTER we know the strategy is balances, to save extra lookup